api/scm.go

```diff
@@ -99,14 +99,19 @@ func SyncRepos(c *gin.Context) {
 		// if repo cannot be captured from GitHub, set to inactive in database
 		if err != nil {
 			repo.SetActive(false)
-			e := database.FromContext(c).UpdateRepo(repo)
-			if e != nil {
+			err := database.FromContext(c).UpdateRepo(repo)
+			if err != nil {
 				retErr := fmt.Errorf("unable to update repo for org %s: %w", org, err)
 
 				util.HandleError(c, http.StatusInternalServerError, retErr)
 
 				return
 			}
+			err = updateUserFavorites(c, repo)
```

Comment:
I think we're missing a space here:

```suggestion
			}

			err = updateUserFavorites(c, repo)
```